packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -610,6 +610,31 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }
   }
 
+  /// Repeatedly calls [pump] with the given `duration` until there are no longer any frames scheduled.
```

Comment:
Looks like this documentation is already getting pulled in from the superclass, see https://master-api.flutter.dev/flutter/flutter_test/WidgetTester/pumpAndSettle.html. I don't think we need to repeat it here.

Reply:
Alright! Got it